staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4131,8 +4134,86 @@ type Event struct {
 	// Type of this event (Normal, Warning), new types could be added in the future
 	// +optional
 	Type string `json:"type,omitempty" protobuf:"bytes,9,opt,name=type"`
+
+	// Time when this Event was first observed.
+	// +optional
+	EventTime metav1.MicroTime `json:"eventTime,omitempty" protobuf:"bytes,10,opt,name=eventTime"`
```

Comment:
The thing is we have 'old' API which we need to fill. IIUC annotations are for adding 'new' data, not for preserving old. It's possible to do it if we create a new API and move the storage from Core to the new API group. I'm not convinced it's better in any way that current solution.